src/tools/ips.rs

```diff
@@ -8,4 +8,31 @@
 /// for MacOS:
 /// sudo python3 ./tools/ips.py --subnet 1.1.1.0/24 --file src/tools/ips.rs --dev lo0
 /// For more information read the documentation of the ips.py script.
-pub const IPS: &[&str] = &[];
+use std::fs;
+
+use serde::Deserialize;
+
+const IPS_LIST_PATH: &str = "./tools/ips_list.json";
+
+#[derive(Default, Clone, Deserialize, Debug)]
+struct IpsList {
+    pub nodes: Vec<String>,
+}
+
+/// Read in the node array generated by the ips.py
+/// script.  If file does not exist, return an empty
+/// array.
+fn load_ips_nodes(filepath: &str) -> Vec<String> {
+    let result = fs::read_to_string(filepath);
+    if result.is_err() {
+        println!("no file {filepath}");
+        return Vec::<String>::new();
+    }
```

Comment:
hm, let's rather fail (panic with `expect`) here directly, no need to be nice 😃

if the file is not found, maybe leave a message to the user to check if he needs to run our script in our Readme documentaiton

Reply:
done